hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/batch/CompositeBatchPersister.java

```diff
@@ -47,6 +52,19 @@ public class CompositeBatchPersister implements BatchPersister {
     private final CommonParserProperties properties;
     private final UpsertQueryGeneratorFactory upsertQueryGeneratorFactory;
 
+    public CompositeBatchPersister(DataSource dataSource, MeterRegistry meterRegistry,
+                                   CommonParserProperties properties,
+                                   UpsertQueryGeneratorFactory upsertQueryGeneratorFactory,
+                                   @Qualifier(TRANSACTION_HASH_BATCH_PERSISTER) Optional<BatchPersister> transactionHashV1BatchPersister) {
```

Comment:
If we have to couple to a transaction hash specific bean name we might as well couple directly to the class name and avoid the qualifier.
```suggestion
Optional<TransactionHashBatchInserter> transactionHashV1BatchPersister) {
```